src/playback/view/internal/inputresourceitem.cpp

```diff
@@ -46,6 +48,9 @@ static const QString VST_MENU_ITEM_ID("VST3");
 static const QString SOUNDFONTS_MENU_ITEM_ID = muse::qtrc("playback", "SoundFonts");
 static const QString MUSE_MENU_ITEM_ID("Muse Sounds");
 
+static const QString GET_MORE_SOUNDS_ID("getMoreSounds");
+static const QString URL_PARAMS("muse-sounds?utm_source=mss-mixer&utm_medium=mh&utm_campaign=mss-mixer-ms-mainpage");
```

Comment:
let's move it to handleMenuItem method because we use it only there

Reply:
Done 👍